metricbeat/metricbeat.yml

```diff
@@ -46,6 +46,14 @@ metricbeat.modules:
   period: 10s
   processes: ['.*']
 
+#------------------------------- kubelet Module ------------------------------
```

Comment:
You can use `short_config: false` in the module fields.yml so it is not added to the short config.